client/reader/search-stream/style.scss

```diff
@@ -319,14 +319,14 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 
 .search-stream__header .section-nav {
 	background: inherit;
-	border-bottom: 2px solid lighten( $gray, 30% );
+	border-bottom: 1px solid lighten( $gray, 20% );
 	box-shadow: none;
-	height: 54px;
+	height: 53px;
```

Comment:
No, had to limit height of `.section-nav`, otherwise, the underline for active tab won't sit perfectly on the gray line:
![screenshot 2017-06-19 14 51 44](https://user-images.githubusercontent.com/4924246/27307510-e2449034-54fe-11e7-9c86-e5cfe51426a1.png)